hw/xbox/xbox.c

```diff
@@ -150,7 +150,7 @@ static void xbox_flash_init(MemoryRegion *rom_memory)
     const char *bootrom_file = object_property_get_str(qdev_get_machine(),
                                                        "bootrom", NULL);
 
-    if ((bootrom_file != NULL) && *bootrom_file) {
+    if ((bootrom_file != NULL) && *botrom_file) {
```

Comment:
?

Reply:
I made the edit from my cellphone, this change was unintended. Will fix asap